learningequality/studio
# Fixes `content_id` bug whenever underlying content is changed :tada:

## Summary

This PR fixes a long standing bug on studio wherein copied contentnodes `content_id` were not changed upon modification of their underlying content. 

Particulary, exercises and file contentnodes were affected by this. Both the cases have been resolved by this PR. 

### Manual verification steps performed
1. Create a channel and a exercise contentnode with multiple assessment items.
2. Copy that exercise node.
3. Make sure the `content_id` of copied node is same as of the original node.
4. Now, try adding, editing or deleting any of the assessment items in the copied node, the `content_id` should be changed!
5. Repeat the following with the file contentnode type as well. That is, try modifying the uploaded file of a copied file node, you should see `content_id` updated.

___
## Reviewer guidance


## References
Closes https://github.com/learningequality/studio/issues/1055
Closes https://github.com/learningequality/studio/issues/3112
Unblocks https://github.com/learningequality/studio/issues/3725



----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
